.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A concise description of what the bug is.
+
+**Steps to reproduce**
+
+**Expected outcome**
+A concise description of what you expected to happen.
+
+**Environment**
+
+* App Mesh controller version
+* Are you using any integrations? X-ray, Jaeger etc. If so versions?
+* Kubernetes version
+* Using EKS (yes/no), if so version?
```

Comment:
Envoy Version and if they are using a custom envoy image would also help.

Reply:
++ good point, let me add it